backend/src/main/java/com/votogether/domain/category/contorller/CategoryController.java

```diff
@@ -29,4 +31,12 @@ public ResponseEntity<List<CategoryResponse>> getAllCategories() {
         return ResponseEntity.status(HttpStatus.OK).body(categories);
     }
 
+    @Operation(summary = "선호 카테고리 추가하기", description = "선호하는 카테고리를 선호 카테고리 목록에 추가한다.")
+    @ApiResponse(responseCode = "201", description = "추가 성공")
+    @PostMapping("/{categoryId}/like")
+    public ResponseEntity<Void> addFavoriteCategory(final Member member, final Long categoryId) {
```

Comment:
```suggestion
    public ResponseEntity<Void> addFavoriteCategory(final Member member, @PathVariable final Long categoryId) {
```
`Q` : `@PathVariable` 어노테이션이 필요하지 않나요?